main.py

```diff
@@ -58,7 +103,7 @@ def pause_on_captcha(driver):
     """If we got redirected to a Captcha, pause and notify for a human to solve."""
     driver.implicitly_wait(pageload_pause)
     if captcha_present(driver):
-        notify('Captcha detected, please solve to continue...')
+        print("Captcha Detected, Please solve to continue...")
```

Comment:
Having used this tool for a little while, my experience is that I now get served captchas while in the search loop at the end of the login process. Therefore, changing this notify to a print seems disadvantageous now. Before I only got them once on initialising the tool, and this would make sense if that were still the case :)

Reply:
ammended